src/all/xinmeitulu/src/eu/kanade/tachiyomi/extension/all/xinmeitulu/Xinmeitulu.kt

```diff
@@ -66,10 +92,28 @@ class Xinmeitulu : ParsedHttpSource() {
 
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
         setUrlWithoutDomain(document.selectFirst("link[rel=canonical]")!!.attr("abs:href"))
-        title = document.select(".container > h1").text()
-        description = document.select(".container > *:not(div)").text()
+        title = document.select(".container > h1").text().trim()
```

Comment:
```suggestion
        title = document.select(".container > h1").text()
```
Jsoup already trims the text.